src/bin/code-suggester.ts

```diff
@@ -94,6 +94,32 @@ yargs
       type: 'boolean',
     },
   })
+  .command(REVIEW_PR_COMMAND, 'Review an open pull request', {
```

Comment:
You should be able to describe all of the commands options inside its own JavaScript file, rather than here, see:

https://github.com/yargs/yargs/blob/master/docs/advanced.md#providing-a-command-module

Reply:
Filing https://github.com/googleapis/code-suggester/issues/133 to clean up separately as it actually appears non-trivial with typing.